notifications/connecting-to-slack.md

```diff
@@ -0,0 +1,50 @@
+# Connecting to Slack
+
+You can connect Aqueduct to your Slack workspace in order to receive automated notifications on your workflow execution status. You can simply click the Slack icon on your [integrations page](../integrations/adding-an-integration/README.md). Aqueduct will ask for [#the bot token](connecting-to-slack#setting-up-slack), [#channels](connecting-to-slack#configuring-channels), and [#the severity level](connecting-to-slack#configuring-default-notification-settings) at which to send slack alerts.
+
+## Setting up Slack
+To connect Aqueduct to slack to recieve slack notifications, you would first need to set up your own Slack App. You can follow these steps to create one:
+* Go to [Slack Apps management page](https://api.slack.com/apps).
+* Hit **Create New App**.
+<figure><img src="../.gitbook/assets/connecting_slack_create_app.png" alt=""><figcaption></figcaption></figure>
+
+* Select **Create from manifest** on the dialog.
+* Select the workspace where you'd like to receive notifications.
+* Choose **yaml** and copy over the following content to the manifest. You can change **name** and **display_name** fields. This step sets up minimum required permissions and configurations for this app.
+    ```yaml 
+    display_information:
+        name: my-aqueduct-bot
+    features:
+        bot_user:
+            display_name: my-aqueduct-bot
+            always_online: false
+    oauth_config:
+        scopes:
+            bot:
+            - channels:join
+            - chat:write
+            - chat:write.public
+            - channels:read
+    settings:
+        org_deploy_enabled: false
+        socket_mode_enabled: false
+        token_rotation_enabled: false
+    ```
+* Once created, you will enter the app's management page. Click **Install to Workspace** and follow the instructions. This completes the Slack App setup.
+* You can also use an existing app. Our minimum permission requirements are: `channels:join`, `chat:write`, `chate:write.public`, and `channels:read`.
```

Comment:
I still think we should remove this. It's quite confusing, and I don't see why I'd want to receive Aq. notifications from another app.

Reply:
Okay let's just remove, we can always add it back if we see user asking questions :)